From bf6db2bc2e8eb1014312f0aab2027610a8abaeba Mon Sep 17 00:00:00 2001
From: Francisco Boni <boboniboni@gmail.com>
Date: Sat, 25 Dec 2021 19:51:15 -0300
Subject: [PATCH] Fix libadwaita req

---
 src/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/meson.build b/src/meson.build
index 7dfe6868..b82d4842 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -122,7 +122,7 @@ easyeffects_deps = [
 	dependency('libpipewire-0.3', version: '>=0.3.31'),
 	dependency('glib-2.0', version: '>=2.56'),
 	dependency('gtk4', version: '>=4.2.1'),
-	dependency('libadwaita-1', version: '>=1.0.0.alpha.4'),
+	dependency('libadwaita-1', version: '>=1.0.0'),
 	dependency('sigc++-3.0', version: '>=3.0.6'),
 	dependency('lilv-0', version: '>=0.22'),
 	dependency('lv2', version: '>=1.18.2'),
-- 
2.34.1

